modules/geo-layers/src/tile-layer/tile-2d-traversal.js

```diff
@@ -0,0 +1,132 @@
+import {CullingVolume, Plane, AxisAlignedBoundingBox} from '@math.gl/culling';
+
+const TILE_SIZE = 512;
+// number of world copies to check
+const MAX_MAPS = 3;
```

Comment:
Should checking world copies be configurable? If I turn off `repeat: true`

![image](https://user-images.githubusercontent.com/15164633/82254628-69191780-9910-11ea-9e80-42f8f1dbf692.png)

The `onViewportLoad` callback in this viewport still holds 75 tiles, while I only count 15 tiles in the viewport. Anecdotally, it seems that many more tiles are loaded when looking west than when looking north.